controls/calendar/navigation.md

```diff
@@ -10,7 +10,7 @@ slug: calendar-navigation
 
 You can navigate between the different views of the Calendar programmatically or let the user change the views from the UI.
 
-## Navigating through Buttons
+## Navigating Through Buttons
```

Comment:
[Never capitalize prepositions](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).
```suggestion
## Navigating through Buttons
```